### DISCUSSION TITLE
Recommended way to use Google Analytics 4 on Nuxt 2 with vue-gtag for SSR

### DISCUSSION THREAD

**Discussion by Coinhexa** [MatteoGabriele/vue-gtag#575] [Q&A]
Basically the title, the documentation doesn't mention anything on how to go about this and while this is referenced in a few issues, the answers are very conflicting

**Comment by MatteoGabriele** [comment]
I started taking care of vue-gtag version 3 that includes a rewrite in typescript and I was also thinking on creating a nuxt module as well. I will hopefully manage to work on that after version 3 is out
